.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,16 +2,17 @@
 Title
 ---
 
-**Important note**
+**Important notes**
 Please note that only PrestoN is able to merge Pull Requests into master.
```

Comment:
```suggestion
**Important note**
```
We should probably remove this line at this point given that it is both inaccurate and does not provide salient information.